pyne/mcnp.py

```diff
@@ -1325,8 +1325,9 @@ def mats_from_inp(inp):
         # information is stored in a dictionary where:
         # key = material number, value = list of densities
         if len(line.split()) > 3:
-            if line.split()[0].isdigit() is True and \
-                    line.split()[1].isdigit() is True and \
+            if line.split()[0].isdigit() and \
+                    line.split()[1].isdigit() and \
+                    not line.split()[2][0].isalpha() and \
```

Comment:
Maybe this test should get split up into its own function? It is now 5 lines long.